src/main/java/frc/robot/Constants.java

```diff
@@ -53,19 +53,118 @@ public static final class DriveConstants {
     public static final double kBackRightChassisAngularOffset = Math.PI / 2;
 
     // SPARK MAX CAN IDs
-    public static final int kFrontLeftDrivingCanId = 13;
-    public static final int kRearLeftDrivingCanId = 15;
-    public static final int kFrontRightDrivingCanId = 17;
-    public static final int kRearRightDrivingCanId = 11;
+    public static final int kFrontLeftDrivingCanId = 12; //13
```

Comment:
What are all these commented out IDs? If not needed remove comment.

Reply:
these are the ids for the old chassis. kept in there until we make a new branch just for that old chassis